kernel/mod_typing.ml

```diff
@@ -107,8 +109,10 @@ let rec check_with_def env struc (idl,(c,ctx)) mp equiv =
 	       let c' = Mod_subst.force_constr cs in
 	       let cst' = Reduction.infer_conv env' (Environ.universes env') c c' in
 	        cst'
-	  in
-	    if not (Univ.Constraint.is_empty cst) then
+     | Primitive _ ->
```

Comment:
Indentation looks wrong.

Reply:
Re-indented the whole function, it was containing tabs.